source/scripting/WeaponComponent.cs

```diff
@@ -3,6 +3,25 @@
 
 namespace GTA
 {
+	public enum ComponentAttachmentPoint : uint
```

Comment:
This enum is missing the `u` suffix for the hashes, which seems to be the convention for hash enums.